.env.dist

```diff
@@ -14,16 +14,8 @@ HYDRA3PK="0xa700000000000000000000000000000000000000000000000000000000000000" #
 # Chain ID (local: 1337, testnet: 9000, devnet: 12000)
 CHAINID="9000"
 
-# RPC endpoints
-CYPRUS1URL="http://rpc.sandbox.quai.network/cyprus1"
-CYPRUS2URL="http://rpc.sandbox.quai.network/cyprus2"
-CYPRUS3URL="http://rpc.sandbox.quai.network/cyprus3"
-PAXOS1URL="http://rpc.sandbox.quai.network/paxos1"
-PAXOS2URL="http://rpc.sandbox.quai.network/paxos2"
-PAXOS3URL="http://rpc.sandbox.quai.network/paxos3"
-HYDRA1URL="http://rpc.sandbox.quai.network/hydra1"
-HYDRA2URL="http://rpc.sandbox.quai.network/hydra2"
-HYDRA3URL="http://rpc.sandbox.quai.network/hydra3"
+# RPC endpoint
+RPCURL="https://rpc.dev.quai.network"
```

Comment:
This is symantic but I think we should change the `RPCURL` variable name to `RPC_URL` because it makes it easier to read.

Reply:
Changed in https://github.com/dominant-strategies/hardhat-example/pull/43/commits/b267aa2edc09cc1a8cd12ac344d03f1166316882